Juanvvc/c172p-detailed
# Bug 274

Fixes #274 

To test this PR:

1. Switch to the branch: `git fetch` and then `git checkout bug-274`

2. Then start fgfs instance 1:

    `fgfs --airport=FHAW --multiplay=out,10,localhost,5701 --multiplay=in,10,localhost,5702 --aircraft=ufo`

3. And fgfs instance 2:

    `fgfs --airport=FHAW --multiplay=out,10,localhost,5702 --multiplay=in,10,localhost,5701 --aircraft=c172p`

Test that the liveries can be updated (might take a little while to see it on the other fgfs instance) and that the glass reflection effects still work.
